packages/module-plugin/integration-mapping.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const { Encrypt } = require('@friggframework/encrypt');
+
+const schema = new mongoose.Schema(
+    {
+        integration: {
+            type: mongoose.Schema.Types.ObjectId,
+            ref: 'Integration',
+            required: true,
+        },
+        sourceId: { type: String }, // Used for lookups
```

Comment:
unique: true? maybe...

Reply:
ah yes, maybe it makes sense to have index on { integration, sourceId }?

Reply:
It definitely does/we'd want it for our case, and I think we'd want it for most cases where volume of mappings accrues over time